app/code/Mediotype/Employee/Api/EmployeeRepositoryInterface.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * @author    Mediotype Development <diveinto@mediotype.com>
+ * @copyright 2019 Mediotype. All Rights Reserved.
+ */
+
+namespace Mediotype\Employee\Api;
+
+/**
+ * This is the Employee CRUD interface. This interface is used as an entry point for interacting with the
+ * Employee data entity.
+ *
+ * Interface EmployeeRepositoryInterface
+ * @package Mediotype\Employee\Api
+ * @api
+ */
+interface EmployeeRepositoryInterface
+{
+    /**
+     * save is used to persist an employee to a database, it accepts an EmployeeInterface object as an argument.
```

Comment:
```suggestion
     * Persist an employee to the data store
```

Short descriptions should be in command format.  It should be exactly what calling the method would do.

The rest of the description is unnecessary, as it's a duplication of what is already documented.

Reply:
To be fair, I told Matt to be verbose in his comments simply because this is a training module and not an actual module we'd release into the wild.

Reply:
That is fair.  Probably not verbosity which is just repetition though - a comment saying that it takes the type specified in param isn't useful?

Otherwise spelling out explicitly what its doing should be a long description underneath the short description - we can satisfy both criteria

Reply:
I like it.